adding toolbar button to display trails of cells deaths

prakashujjwal1010 opened the pull request:
This PR
- adds toolbar button to display trails of cells deaths
- adds the state of button to datastore
- modified tutorial.js and locale.ini for new toolbar button 
- adds attribution for new icon used

![Screenshot from 2020-01-07 00-28-17](https://user-images.githubusercontent.com/40318927/71841595-cd67d700-30e5-11ea-9255-c826b7e387be.png)

![Screenshot from 2020-01-07 00-28-20](https://user-images.githubusercontent.com/40318927/71841613-d22c8b00-30e5-11ea-82c5-b2422eac43d6.png)

llaske commented:
Very cool.

Two UI remarks:

- I think the death color could be brighter, may be a lighter gray or why not do a very light version of the stroke color
- I'm not a big fan of your icon: it's too big (icons should have margin between the border and the content) and it's not really explicit. May be you could find something around the concept of "footsteps"?

prakashujjwal1010 commented:
- changed icon (also added the attribution)
![Screenshot from 2020-01-07 23-38-08](https://user-images.githubusercontent.com/40318927/71917817-fdc37a00-31a6-11ea-9fcb-257c49d65d09.png)

![Screenshot from 2020-01-07 23-32-30](https://user-images.githubusercontent.com/40318927/71917813-f9975c80-31a6-11ea-816b-c025ae552f97.png)
![Screenshot from 2020-01-07 23-36-49](https://user-images.githubusercontent.com/40318927/71917876-17fd5800-31a7-11ea-9546-b00206a6b173.png)

i have not use lighter version of stroke color as if stoke color is like yellow or any other bright color, then we can have a problem where both are alive and dead cells are of same color.

llaske commented:
Good job. Thanks.
